src/collections/_documentation/platforms/native/index.md

```diff
@@ -194,6 +194,33 @@ initialization:
   sentry_options_set_release(options, "my-unique-release-1.0.0");
 ```
 
+### Release Health
+
+Monitor the [health of releases](/workflow/releases/health/) by observing user adoption, usage of the application, percentage of [crashes](/workflow/releases/health/#crash), and [session data](/workflow/releases/health/#session). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
+
+To benefit from the health data you must use at least version 0.4.0 of the Native SDK.
+The collection of release health metrics is enabled by default unless specifically disabled in the initialization options of the SDK.
+
+```c
+sentry_options_t *options = sentry_options_new();
+// Session tracking is enabled by default. To disable:
+sentry_options_set_auto_session_tracking(options, 0);
+sentry_init(options);
+```
+
+The SDK automatically starts a new session when it is initialized via `sentry_init`, and will end that session automatically when the SDK is shut down via `sentry_shutdown`, or explicitly by calling `sentry_end_session`.
+The SDK currently only tracks one concurrent session and will also end the running session when using `sentry_start_session`.
+
+For more details, see the [full documentation on Release Health](/workflow/releases/health/).
+
+### Identification of the User
+
+By default, the SDK will try to use the `id`, `email` or `username` properties of the object set via `sentry_set_user` as the sessions distinct identifier. While this ID does contain personally identifiable information and is transmitted to the sentry backend, it will not be persisted server-side.
```

Comment:
nit: add a comma after `email`

nit: session's distinct identifier

Suggest: While this ID contains personally identifiable information, and is transmitted to the Sentry backend, the ID will not persist server-side.